### DISCUSSION TITLE
[Esame 22 gennaio 2024] Esercizio 2

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#443] [Esami M-Z]
![22 gennaio es. 2](https://github.com/user-attachments/assets/8bc6ebe0-8901-41cb-8d63-8c5354c2ec55)

<details>
  <summary>soluzione del prof</summary>
  <br>
    <img src="https://github.com/user-attachments/assets/fa54541a-1d0b-4bb5-b137-69eb0a245a43" alt="soluzione"  />
    
</details>

**Comment by thegeek-sys** [comment]
![immagine](https://github.com/user-attachments/assets/b7aa4faf-9f66-408d-a014-c2b303266784)
